.vib/rclone/goss/rclone.yaml

```
@@ -0,0 +1,4 @@
+command:
+  check-rclone-sync:
+    exec: rclone sync /opt/bitnami/rclone/bin /tmp/RANDOM_FOLDER && [ -f /tmp/RANDOM_FOLDER/rclone ]
```

We always try to create our tests so they are retry-able and independent from each other. Here the test would run without issues even if we repeat the same folder, but it is always good to follow the best practices. In this type of situation, we normally randomise the folder name like this:

```suggestion
    {{- $rnd_folder := printf "folder_%s" (randAlpha 5) }}
    exec: rclone sync /opt/bitnami/rclone/bin /tmp/{{ $rnd_folder }} && [ -f /tmp/{{ $rnd_folder }}/rclone ]
```

You are going to kill me, but my suggestion was more of a sketch than final code. Sorry about that 😓

---

Ah! My bad!
Let me fix it real quick